qhub/utils.py

```diff
@@ -349,3 +349,15 @@ def deep_merge(*args):
         return [*d1, *d2]
     else:  # if they don't match use left one
         return d1
+
+
+def pip_install_qhub(qhub_version: str) -> str:
```

Comment:
good catch :+1:

Reply:
@costrouc this is unsophisticated way of handling the qhub_version issue for now. Would this be okay to add?

Reply:
Yeup this works with me.